pnpm-lock.yaml

```diff
@@ -5173,6 +5185,12 @@ packages:
     cpu: [arm64]
     os: [android]
 
+  '@esbuild/android-arm64@0.24.2':
```

Comment:
If we can get all of the other templates to v6 that would be great then these esbuild dependencies should collapse